packages/accounts/src/kernel-zerodev/e2e-tests/constants.ts

```diff
@@ -1,2 +1,2 @@
-export const MUMBAI_RPC_URL = process.env.MUMBAI_RPC_URL;
```

Comment:
not sure when we did this - but putting it back to api key only. we should hard code the rpc url in code.